GPU/GPUTracking/Global/GPUChainTrackingTRD.cxx

```diff
@@ -52,7 +53,7 @@ int GPUChainTracking::RunTRDTracking()
       if (!Tracker.CheckTrackTRDCandidate(trktrd)) {
         continue;
       }
-
+      GPUWarning("It does not make sense to load input track without time window information, except when processing Run 2 data");
```

Comment:
Also this should depend on whether we process a time frame or not.
The conversion to O2 tracks gets the time0 as:
```
time0 = tracks[i].GetParam().GetTZOffset();
```
The problem is that the deltaTFwd and deltaTBwd is not so obvious.
For this the tracker iterates over the TPC clusters. Perhaps for now you can just set 1 TPC drift time? That should at least work. And this mode of operation is anyway mostly for tests, the default will be using the o2 track I guess.

Reply:
Does `GetTZOffset()` have a similar precision as the time of the `TrackTPC`? At least when running with a single thread and looking at the times of the GPU and O2 track types they did not really match (for the same track index). And then in O2 I check if the track is on the A/C side or if it crosses the CE. In the latter case the time should be precise and I check only one TRD trigger time. Trying to do the same now with the GPU track types I find much less tracks than with the O2 track version (roughly factor 10)

Reply:
I had the wrong unit for the TPC drift time... This works now as expected